Gopkg.toml

```diff
@@ -24,7 +24,7 @@
 
 [[constraint]]
   name = "github.com/opencontainers/runtime-spec"
-  revision = "4e3b9264a330d094b0386c3703c5f379119711e8"
+  revision = "5806c35637336642129d03657419829569abc5aa"
```

Comment:
In the introduced spec changes, I don't see any hooks related change. Why do you need to update runtime-spec?

Reply:
Because of this: https://github.com/kata-containers/agent/blob/master/Gopkg.toml#L19

```
$ sed -i 's/7e8e20b10b71fe3044a24175b8a686421e9d2c24/03f040f14dceb858f6411db275fcac77f9f42a8d/' Gopkg.toml
$ dep ensure -update github.com/kata-containers/agent
Solving failure: No versions of github.com/kata-containers/agent met constraints:
        03f040f14dceb858f6411db275fcac77f9f42a8d: Could not introduce github.com/kata-containers/agent@03f040f14dceb858f6411db275fcac77f9f42a8d, as it has a dependency on github.com/opencontainers/runtime-spec with constraint 5806c35637336642129d03657419829569abc5aa, which has no overlap with existing constraint 4e3b9264a330d094b0386c3703c5f379119711e8 from (root)
[...]
```